### DISCUSSION TITLE
Is Paste available to use for non Twilio UX Designers?

### DISCUSSION THREAD

**Discussion by janakleitsch** [twilio-labs/paste#1867] [Q&A] [answered]
Hi, the company I work at is building our product on top of Twilio Flex. Is Paste available for our team to use as well? Thanks!

**Comment by gloriliale** [comment, marked as answer]
Hi @janakleitsch! That's very exciting, welcome! And yes! This is exactly what we're building Paste for. We have a Figma Community file (https://www.figma.com/@twilio) as well. If you're open to leaving feedback on it, we would appreciate hearing your thoughts.

Do you already have a contact on the Flex support team, or another team, that you'll be working with? Happy building!

**Reply by janakleitsch** [reply]
Hi, thanks for the info. Great news! I do not have a contact on the Flex support team currently. What is the best way to get access? Go through the engineering manager on my team, or request from Twilio?

**Reply by gloriliale** [reply]
We'll be happy to help any way we can. It'd be really great to know more about your use case as well; could you send us an email to paste@twilio.com so that we can learn more?

**Reply by janakleitsch** [reply]
thanks @gloriliale! I emailed the address above with more specifics.
